### ISSUE TITLE
SEA chess games' captured piece silhouettes are messed up.

### ISSUE THREAD

**Issue by PLMMJ-FH** [gbtami/pychess-variants#1650]
For the pawn and rook, their Chess silhouettes are used instead of their Makruk ones in the little capture tracker thing near each player's name. The queen uses a unique silhouette not used anywhere else, and the bishop has the King's appearance.
![MakrukSilhouetteBug](https://github.com/user-attachments/assets/c9464766-5eaf-47d5-bb56-72c40c9dee24)

**Comment by gbtami** [comment]
Can you provide the game link please?

**Comment by PLMMJ-FH** [comment]
No game link is needed, it happens on every single game of Makruk for me.

**Comment by gbtami** [comment]
https://www.pychess.org/A28Nuxd6

**Comment by PLMMJ-FH** [comment]
It happens in Ouk Chatrang too.

**Comment by PLMMJ-FH** [comment]
Nope, still happening, and the King shows being able to capture the Rook for some reason???
![Shatranj King Castling Thing](https://github.com/user-attachments/assets/135fb6b2-d6eb-4b2c-a944-58ba24edea06)

**Comment by gbtami** [comment]
Don't mix unrelated issues please. If you find a new bug create a new issue. Thx.

**Comment by gbtami** [comment]
Fixed via https://github.com/gbtami/pychess-variants/commit/7450969f9beb37589b10b9557f4b9320a905db53
